api/document/tests/end_to_end_test.py

```diff
@@ -17,16 +18,33 @@
 EXAMPLE_DOCS_DIR = MY_DIR / '..' / '..' / 'example_docs'
 
 
-@pytest.mark.xfail(raises=AssertionError, reason="Fix this!!")
+def clean(content: bytes) -> str:
+    s = content.decode('utf-8')
+
+    # We're going to strip out `id` attributes here because our
+    # legacy XML importer seems to auto-assign ids in a different
+    # way than our serializer.
+    s = re.sub(r'id="[a-z0-9_]+', '', s)
+
+    return s
```

Comment:
Mmm. I'd think that this indicates another bug; I suspect the deserializer isn't accounting for the type emblem.

Reply:
Oh blaaaarg indeed, I think this goes back to the ol' #903.